definitions/divorce/json/CaseEvent/CaseEvent-prod.json

```diff
@@ -0,0 +1,17 @@
+[
```

Comment:
please use specific feature toggle. Since when it's a "prod"?

Reply:
updated